blues/buildmol.py

```diff
@@ -0,0 +1,89 @@
+import os
+from IPython.display import Image
+import numpy as np
+from rdkit import Chem
+from rdkit.Chem import AllChem, Draw
+
+
+def order_atoms(atom_file):
+    """
+    Enumerates the .txt file containing all atoms in the molecule, order the same as JANPA output and eliminate whitespace
+    Parameters
+    ----------
+    filename: string of file path
+
+    Returns: ordered list of atoms in molecule
+    """
+    with open(atom_file) as f:
+        atoms = f.readlines()
+        atoms = [atom.strip() for atom in atoms]
+
+    return atoms
+
+
+def add_atoms(atom_file):
+    """
+    add atoms and correct bonds to build molecule as object through RDKit
+    """
+    # create molecule under RWMol class
+    mol = Chem.RWMol()
+    # call function to order atoms
+    atoms = order_atoms(atom_file)
+    # add all atoms to molecule
+    for atom in atoms:
+        mol.AddAtom(Chem.Atom(atom))
+    return mol
```

Comment:
Add in `Chem.SanitizeMol(mol)` before returning `mol`

Can remove it from `draw_molecule`